app/models/user.rb

```diff
@@ -1,11 +1,12 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/Naledi-Dikgale/blog-app/pull/13#discussion_r1274870050), kindly make sure to remove this piece of commented code to make your codebase clean and tidy 📌✨